part3/roles-responsibilities.qmd

```diff
@@ -2,13 +2,220 @@
 
 ## Introduction
 
-Defining team structures and responsibilities for agent operations.
+The emergence of AgentOps represents a fundamental shift in how organizations structure and manage AI-powered systems. It introduces the need for new role definitions, collaborative patterns, and operational frameworks to handle AI agents, non-deterministic behaviors, and multi-agent interactions. This evolution requires both role specialization and cross-functional integration.
+In this section, we will explore key roles and responsabilities within GenAI projects.
 
 ## Key Roles
 
-- Agent Engineers
-- MLOps Engineers
-- Data Scientists
-- Platform Engineers
+Most organisations are consumers of AI models rather than builders. Their primary focus tends to be on integration and orchestration, agent systems, cost management, prompt engineering, and ensuring safety and governance.
 
-*Content to be developed*
\ No newline at end of file
+The key roles listed below relate to how organisations implement GenAI. Because GenAI work requires significant collaboration, the exact division of responsibilities often depends on the organisation’s size. In some cases, one person may take on several roles, or hold a hybrid role that combines responsibilities from different areas.
+
+### Technical Roles
+
+These roles directly build, implement, and maintain AI systems, handling the hands-on technical work and domain expertise required to deliver AI solutions.
+
+#### AI Engineer {.unnumbered}
+An AI Engineer integrates and optimizes pre-trained AI models into production applications, focusing on making AI capabilities work reliably and cost-effectively within business systems rather than building models from scratch.
+
+Primary Responsibilities:
+
+* Integrate and fine-tune existing LLMs with most organizations using pre-trained models via APIs
+* Design and implement RAG (Retrieval Augmented Generation) architectures for domain specific applications
+* Develop prompt engineering strategies and optimize prompt templates for production use
+* Build evaluation and testing frameworks for non-deterministic AI systems, ensuring quality despite variable outputs
+* Optimize for cost and performance on the application logic level using prompt optimization, model selection, workflow design that minimizes API calls and output optimizaion.
+* Create AI agents workflows and orchestrate AI tool chains for complex tasks
```

Comment:
What is an AI agents workflow or tool chain? I agree that part of an AI Engineer's job is to develop tools that give agents access to external data sources, but I think we should cover that before optimizing cost / performance and before eval. This line seems a bit like AI-speak